php/class-customize-snapshot-manager.php

```diff
@@ -1322,15 +1358,97 @@ public function remove_all_non_snapshot_admin_bar_links( $wp_admin_bar ) {
 	 * Underscore (JS) templates for dialog windows.
 	 */
 	public function render_templates() {
+		$data = $this->get_month_choices();
+
+		$tz_string = get_option( 'timezone_string' );
+		if ( $tz_string ) {
+			$tz = new \DateTimezone( $tz_string );
+			$formatted_gmt_offset = $this->format_gmt_offset( $tz->getOffset( new \DateTime() ) / 3600 );
+			$tz_name = str_replace( '_', ' ', $tz->getName() );
+
+			/* translators: 1: timezone name, 2: gmt offset  */
+			$date_control_description = sprintf( __( 'This site\'s dates are in the %1$s timezone (currently UTC%2$s).', 'customize-snapshots' ), $tz_name, $formatted_gmt_offset );
+		} else {
+			$formatted_gmt_offset = $this->format_gmt_offset( get_option( 'gmt_offset' ) );
+
+			/* translators: %s: gmt offset  */
+			$date_control_description = sprintf( __( 'Dates are in UTC%s.', 'customize-snapshots' ), $formatted_gmt_offset );
+		}
 		?>
 		<script type="text/html" id="tmpl-snapshot-preview-link">
 			<a href="#" target="frontend-preview" id="snapshot-preview-link" class="dashicons dashicons-welcome-view-site" title="<?php esc_attr_e( 'View on frontend', 'customize-snapshots' ) ?>">
 				<span class="screen-reader-text"><?php esc_html_e( 'View on frontend', 'customize-snapshots' ) ?></span>
 			</a>
 		</script>
 
-		<script type="text/html" id="tmpl-snapshot-edit-link">
-			<a href="{{ data.editLink }}" id="snapshot-edit-link" class="dashicons dashicons-calendar-alt" title="<?php esc_attr_e( 'Edit Snapshot','customize-snapshots' ); ?>"></a>
+		<script type="text/html" id="tmpl-snapshot-schedule-button">
+			<a href="#" id="snapshot-schedule-button" class="dashicons dashicons-calendar-alt" title="<?php esc_attr_e( 'Schedule Snapshot','customize-snapshots' ); ?>"></a>
+		</script>
+
+		<script type="text/html" id="tmpl-snapshot-schedule">
+			<div id="snapshot-schedule">
+				<div class="snapshot-schedule-title">
+					<h3>
+						<?php esc_html_e( 'Schedule Snapshot', 'customize-snapshots' ); ?>
+						<span class="reset-time">(<a href="#"><?php esc_html_e( 'Reset', 'customize-snapshots' ) ?></a>)</span>
+					</h3>
+					<span class="snapshot-schedule-description">
+						<span class="snapshot-scheduled-countdown"></span>
+						<span class="timezone-info"><?php echo esc_html( $date_control_description ); ?></span>
+					</span>
+					<a href="{{ data.editLink }}" class="dashicons dashicons-edit snapshot-edit-link" aria-expanded="false"></a>
```

Comment:
Yes, it's not specific to scheduling. We can move it to the top but we don't have much space left there.
I will add `target` and `screen-reader-text` and `title`.